src/Pages/IndexPage/Comps/AddNoteBoard.js

```diff
@@ -0,0 +1,92 @@
+import React, { useContext, useEffect, useState } from "react";
+import { makeStyles, fade } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import TextField from "@material-ui/core/TextField";
+import IndexPageContext from "../../../Context/IndexPageContex";
+import { Button } from "@material-ui/core";
+
+import ColorPickerModal from "./ColorPickerModal";
+import usePostNewNote from "../../../Hooks/usePostNewNote";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    display: "flex",
+    flexDirection: "column",
+  },
+
+  input: {
+    borderRadius: 4,
+    position: "relative",
+    backgroundColor: theme.palette.common.white,
+    border: "0px solid",
+    fontSize: 16,
+    width: "auto",
+    padding: "10px 12px",
+    transition: theme.transitions.create(["border-color", "box-shadow"]),
+    // Use the system font instead of the default Roboto font.
+    fontFamily: [
```

Comment:
you already did that,
just move that to the index.css
dont need to repeat in every page